torch/csrc/jit/runtime/profiling_graph_executor_impl.cpp

```diff
@@ -489,6 +490,11 @@ const ExecutionPlan& ProfilingGraphExecutorImpl::getOptimizedPlanFor(
     auto copy = graph->copy();
     runProfilingInsensitiveOptimizations(copy);
     pr_ = ProfilingRecord::instrumentGraph(copy);
+    // TODO: order for now is important; we should make
```

Comment:
What was the context of this ?

Reply:
My bad on this one during rebase.
The comment should be added when we add a second pass to insert profile_ivalue nodes for nvfuser https://github.com/pytorch/pytorch/pull/47668/files#diff-4b6a31b50172ed0fce36027a913fab9e59e496954264ad5e021324c74355a9c2R498-R500

cc'ing @Krovatkin

Reply:
@eellison 
If I remember correctly if we run `InsertProfileNodesForCUDAFuser` first it will insert a profile node on a list input (which is also a graph input) to `aten::grad_sum_to_size` which would in turn break the logic in `InsertProfileNodesForSpecializeAutogradZero` that checks that there's a direct use of the graph input that is also `aten::grad_sum_to_size` : https://github.com/pytorch/pytorch/pull/47667/files#diff-d606514ce9e63ab23356691f3f6df0616222cd50b8a41239add4601f14a53392R16

Ideally, it would be nice if different passes could insert profile nodes in whatever order, but I wasn't quite convinced of the merit of that approach, so I just decided to document it.